### PR TITLE
processAction - Weight Check

### PR DESCRIPTION
**Pull request by tkcjesse**
All the work goes to @BoGuu. I only helped with testing and this pull/commit.

**Please don't merge this yet...**

Essentially prior to this you would basically lose all your shit if you had processed items weigh more than the start item. So this will stop that and only process what you can hold.

We tested this on multi-process all ways.  one-to-one, one-to-many, and many-to-one. Should be good for many-to-many as well.

-"It works by calculating the total weight of all the needed materials and total weight of all the products, and uses the net change to see if you can possibly process anything"

### CODE REVIEW

File: Altis_Life.Altis/stringtable.xml [hunk 1]
```diff
@@ -7277,6 +7277,9 @@
             <Portuguese>Você precisa ficar a menos de 10m para processar.</Portuguese>
             <Polish>Musisz być w odległości 10 m aby przetwarzanie się odbyło.</Polish>
         </Key>
+        <Key ID="STR_Process_Weight">
```
Comment (Jawshy) [inline Altis_Life.Altis/stringtable.xml]: Can you add **all** of the languages.

**Comment by tkcjesse** [comment]
Please don't merge yet.

**Comment by NiiRoZz** [comment]
Can use count instead of forEach when you can for optimization

**Comment by tkcjesse** [comment]
@NiiRoZz if you want to fuck with it further and stuff feel free to merge it. Otherwise it'll wait till 2mor.

**Pull request merged**